package.json

```diff
@@ -6,17 +6,17 @@
   "main": "index.js",
   "scripts": {
     "prepack": "npmignore --auto --commentLines=autogenerated",
-    "prelint": "npm run lint:docs",
+    "prelint": "npm run lint:docs && npm run lint:eslint-docs",
     "lint:docs": "markdownlint \"**/*.md\"",
+    "lint:eslint-docs": "npm run update:eslint-docs --check",
     "lint": "eslint .",
     "postlint": "npm run type-check",
     "pretest": "npm run lint",
     "test": "npm run unit-test",
     "posttest": "aud --production",
     "type-check": "tsc",
     "unit-test": "istanbul cover node_modules/mocha/bin/_mocha tests/lib/**/*.js tests/util/**/*.js tests/index.js",
-    "generate-list-of-rules": "md-magic --path README.md",
-    "generate-list-of-rules:check": "npm run generate-list-of-rules && git diff --exit-code README.md"
+    "update:eslint-docs": "eslint-doc-generator --ignore-config all --url-configs \"https://github.com/jsx-eslint/eslint-plugin-react/blob/master/README.md#shareable-configs\""
```

Comment:
```suggestion
    "update:eslint-docs": "eslint-doc-generator --ignore-config all --url-configs \"https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs\""
```

Reply:
I was informed at one point that the anchor link would only work on mobile if the link pointed to the actual README file: https://github.com/jest-community/eslint-plugin-jest/pull/1226#discussion_r964574555

I'll have to double-check if this is true. If it is true, I'll update the link to use `HEAD` instead of `master`.

Reply:
I just confirmed on my iPhone; it works fine.

Reply:
Updated to the shorter URL.